RevenueCat/Plugins/Editor/RevenueCatDependencies.xml

```diff
@@ -2,10 +2,10 @@
 <dependencies>
     <androidPackages>
         <androidPackage spec="org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.2.61" />
-        <androidPackage spec="com.revenuecat.purchases:purchases:3.1.0" />
-        <androidPackage spec="com.revenuecat.purchases:purchases-hybrid-common:1.1.2" />
+        <androidPackage spec="com.revenuecat.purchases:purchases:3.2.0" />
```

Comment:
I think this one can be removed. The latest `purchases-hybrid-common` should include it. Let's test it before we release but I removed it from Flutter. We can get weird issues if the version that `purchases-hybrid-common` uses differs from the one we declare here.